src/GovUk.Education.ExploreEducationStatistics.Publisher/local.settings.json

```diff
@@ -9,7 +9,10 @@
     "CoreStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://data-storage:10000/devstoreaccount1;",
     "NotificationStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://data-storage:10001/devstoreaccount1;",
     "PublisherStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://data-storage:10000/devstoreaccount1;QueueEndpoint=http://data-storage:10001/devstoreaccount1;TableEndpoint=http://data-storage:10002/devstoreaccount1;",
-    "PublicStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://data-storage:10000/devstoreaccount1;TableEndpoint=http://data-storage:10002/devstoreaccount1;"
+    "PublicStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://data-storage:10000/devstoreaccount1;TableEndpoint=http://data-storage:10002/devstoreaccount1;",
+    "AzureFunctionsJobHost__logging__logLevel__Default": "None",
```

Comment:
I'm not sure that using `"Default: "None"` here and in the two other function projects is a good idea.
Have you tested this? I think if you do you will find you have supressed all of our own service code logging which will log under the `GovUk.Education.ExploreEducationStatistics` namespace and now be filtered out.

In addition to this you are supressing warnings and errors from any other dependencies that fall outside the `Microsoft` namespace. Compare this with the default logging configuration that you are overriding where we currently have `"Default": "Error"`. This means we wouldn't spot them when developing locally.

I would do the detective work to find the source of the log 'spam' that's causing the issue first and then you can target it's namespace to filter it out, if it's all being generated from the same source.

If that's too much then something like this might be better

```
"logging": {
  "logLevel": {
    "default": "Error",
    "Function": "Information",
    "GovUk.Education.ExploreEducationStatistics": "Information",
    "Microsoft": "Warning"
  }
```

Reply:
Talked to Ben about this - switching the default from None to Error. The log spam still isn't on.